tasks/wait-for-node-ready.yml

```
@@ -1,18 +1,18 @@
 ---
 # for master nodes wait for the master API port to be ready
 - block:
-  - name: OCP | Wait for Node Ready | Determine master API port
+  - name: OCP | Wait for Master Ready | Determine master API port
     set_fact:
       openshift_master_api_port: 443
     when: openshift_master_api_port is not defined
 
-  - name: OCP | Wait for Node Ready | Wait for master API to come back online
+  - name: OCP | Wait for Master Ready | Wait for master API to come back online
     wait_for:
       state: started
       delay: 10
       port: "{{ openshift_master_api_port }}"
       timeout: 600
-  when: inventory_hostname in groups['masters']
+    when: inventory_hostname in groups['masters']
```

This when statement has been indented. It now only applies to the `wait_for` instead of the the entire block. I think this change should be reverted.

```suggestion
  when: inventory_hostname in groups['masters']
```

---

Reverted change. I believe I was thinking here that the block was on the entire task, which its not.